src/core/server/saved_objects/serialization/index.ts

```diff
@@ -77,12 +81,15 @@ function assertNonEmptyString(value: string, name: string) {
   }
 }
 
-/** @internal */
+/** @public */
```

Comment:
Should we explain why you would need to use this?

Reply:
👍 Wanted to keep all documentation for next PR, but the `serializer` changes are already exposed as public in this PR. Thanks.